chef/cookbooks/ceph/recipes/mon.rb

```diff
@@ -19,8 +19,9 @@
 include_recipe "ceph::conf"
 
 service_type = node["ceph"]["mon"]["init_style"]
+address = node["crowbar"]["network"]["public"] || node["hostname"]
```

Comment:
A couple of things are wrong here :-)

- I don't think you want to hardcode the public network, but reference one defined in the client_network attribute (https://github.com/crowbar/crowbar-ceph/blob/master/chef/data_bags/crowbar/template-ceph.json#L19).

- Do not use `node["crowbar"]["network"]["public"]`, but instead `Barclamp::Inventory.get_network_by_type(node, "public").address`. This is important as the location of the IP address could change.